pysph/examples/dam_break_2d.py

```diff
@@ -26,31 +32,40 @@
 h = 0.039
 g = 9.81
 ro = 1000.0
-co = 10.0 * np.sqrt(2*9.81*fluid_column_height)
+co = 10.0 * np.sqrt(2 * 9.81 * fluid_column_height)
 gamma = 7.0
 alpha = 0.1
 beta = 0.0
-B = co*co*ro/gamma
+B = co * co * ro / gamma
 p0 = 1000.0
 
 
 class DamBreak2D(Application):
+
     def add_user_options(self, group):
         self.hdx = 1.3
         group.add_argument(
             "--h-factor", action="store", type=float, dest="h_factor",
             default=1.0,
             help="Divide default h by this factor to change resolution"
         )
+        group.add_argument(
+            "--kernel-corr", action="store", type=str, dest='kernel_corr',
+            default='', help="Type of Kernel Correction"
+        )
```

Comment:
You can give a set of choices for this argument.  See this for example: https://github.com/pypr/pysph/blob/master/pysph/solver/application.py#L458

Reply:
Modified this